src/pages/workspace/accounting/qbo/advanced/QuickbooksAdvancedPage.tsx

```diff
@@ -0,0 +1,177 @@
+                >
+                    <ScreenWrapper
+                        includeSafeAreaPaddingBottom={false}
+                        shouldEnableMaxHeight
+                        testID={QuickbooksAdvancedPage.displayName}
+                    >
+                        <HeaderWithBackButton title={translate('workspace.qbo.advancedConfig.advanced')} />
+
+                        <ScrollView contentContainerStyle={[styles.ph5, styles.pb5]}>
+                            {qboToggleSettingItems.map((item) => (
+                                <ToggleSettingOptionRow
+                                    key={item.title}
+                                    errors={item.errors}
+                                    onCloseError={item.onCloseError}
+                                    title={item.title}
+                                    subtitle={item.subtitle}
+                                    shouldPlaceSubtitleBelowSwitch
+                                    wrapperStyle={styles.mv3}
+                                    isActive={item.isActive}
+                                    onToggle={item.onToggle}
+                                    pendingAction={item.pendingAction}
+                                />
+                            ))}
+
+                            <View style={styles.mv3}>
+                                <SpacerView
+                                    shouldShow
+                                    style={[styles.chatItemComposeBoxColor]}
+                                />
+                            </View>
+
+                            <ToggleSettingOptionRow
+                                title={translate('workspace.qbo.advancedConfig.reimbursedReports')}
+                                errors={ErrorUtils.getLatestErrorField(qboConfig ?? {}, CONST.QUICK_BOOKS_CONFIG.COLLECTION_ACCOUNT_ID)}
+                                onCloseError={() => Policy.clearQBOErrorField(policyID, CONST.QUICK_BOOKS_CONFIG.COLLECTION_ACCOUNT_ID)}
+                                subtitle={translate('workspace.qbo.advancedConfig.reimbursedReportsDescription')}
+                                shouldPlaceSubtitleBelowSwitch
+                                wrapperStyle={styles.mv3}
+                                pendingAction={pendingFields?.collectionAccountID}
+                                isActive={isSyncReimbursedSwitchOn}
+                                onToggle={() =>
+                                    Connections.updatePolicyConnectionConfig(
+                                        policyID,
+                                        CONST.POLICY.CONNECTIONS.NAME.QBO,
+                                        CONST.QUICK_BOOKS_CONFIG.COLLECTION_ACCOUNT_ID,
+                                        isSyncReimbursedSwitchOn ? '' : selectedAccount,
+                                    )
+                                }
+                            />
+
+                            {collectionAccountID && (
```

Comment:
```suggestion
                            {!!collectionAccountID && (
```
Conditions should be boolean. This is causing a console error